IDE/Espressif/ESP-IDF/examples/wolfssl_server/main/Kconfig.projbuild

```diff
@@ -0,0 +1,29 @@
+menu "Example Configuration"
```

Comment:
Is this right for in the wolfssl_server directory? Looks like benchmark options.

Reply:
Good catch! That's correct - it was a copy of `Kconfig.projbuild` with the intention of having all the examples have kconfig capabilities. I've fixed that, along with other polishing updates to the examples. Once I've confirmed all is well, I'll push an additional commit.